zircon.core/src/commonMain/kotlin/org/hexworks/zircon/internal/component/renderer/decoration/BoxDecorationRenderer.kt

```diff
@@ -47,13 +50,24 @@ data class BoxDecorationRenderer(
                 } else {
                     finalTitle
                 }
+                val titleOffsetX = when {
+                    titleAlignment.isLeft() -> 0
+                    titleAlignment.isRight() -> size.width - cleanText.length - 4
+                    titleAlignment.isCenter() -> (size.width - cleanText.length - 4) / 2
+                    else -> throw IllegalStateException("unreachable")
```

Comment:
You should have written `this should never happen` 😂

Reply:
You want me to? 😛

Reply:
Haha, it is OK.